graphene_django_optimizer/query.py

```diff
@@ -271,7 +271,7 @@ def _get_name_from_resolver(self, resolver):
             return 'id'
         elif isinstance(resolver, functools.partial):
             resolver_fn = resolver
-            if resolver_fn.func == DjangoListField.list_resolver:
+            if isinstance(resolver_fn, functools.partial) and resolver_fn.func != default_resolver:
```

Comment:
Looks fine but we are doing the same check twice `isinstance(resolver, functools.partial)` except with another name. Could you change this (and the if below as well thus with a `else`)? 👍

Reply:
I've updated this, we can't reduce this to an `if` .. `else` though as the second check needs to be performed even if the first case is true so I've just removed the redundant type check.

Reply:
True I didn't see it was writing a new value 👀 Thank you!